src/rt/sections_elf_shared.d

```diff
@@ -461,8 +461,6 @@ extern(C) void _d_dso_registry(CompilerDSOData* data)
         else
         {
             // static DSOs are unloaded in reverse order
-            assert(pdso._tlsSize == _tlsRanges.back.length);
-            _tlsRanges.popBack();
```

Comment:
Just to check - this is called once per the executable, not once per thread, right? Meaning, `_tlsRanges` will have just one element.